packages/components/src/docs/Migration/ui-select.mdx

```diff
@@ -0,0 +1,242 @@
+# Migration Guide: From `ui-select` to `sd-select`
+
+The new `sd-select` is designed to replace the `ui-select`. Instead of mainly providing content via attributes, the `sd-select` component uses slots to allow for more flexibility and customization.
+
+## 💾 Slots
+
+### ✨ New Slots
+
+#### label
```

Comment:
```suggestion
#### [label]
```

Reply:
pls change this in all level 4 headings as in the other migration guides. This improves readability and follows the structure of the other guides in this chapter.